config.go

```diff
@@ -28,15 +33,15 @@ type RefreshPeriod struct {
 }
 
 type Config struct {
-	CacheDir        string          `yaml:"cache_dir"`
-	Port            int             `yaml:"port"`
-	Repos           map[string]Repo `yaml:"repos,omitempty"`
-	PurgeFilesAfter int             `yaml:"purge_files_after"`
-	DownloadTimeout int             `yaml:"download_timeout"`
-	Prefetch        *RefreshPeriod  `yaml:"prefetch"`
-	HttpProxy       string          `yaml:"http_proxy"`
-	UserAgent       string          `yaml:"user_agent"`
-	LogTimestamp    bool            `yaml:"set_timestamp_to_logs"`
+	CacheDir        string           `yaml:"cache_dir"`
+	Port            int              `yaml:"port"`
+	Repos           map[string]*Repo `yaml:"repos,omitempty"`
```

Comment:
Is there a reason to switch to pointers? Is it efficiency related?

Reply:
It's so the repo record can be updated directly, rather than by writing to the config map.

Reply:
Makes sense :+1: